src/flint/fmpq_poly.jl

```diff
@@ -530,6 +530,29 @@ function is_square_with_sqrt(x::QQPolyRingElem)
   return true, divexact(s, s1)
 end
 
+################################################################################
+#
+#  Interpolation
+#
+################################################################################
+
+function interpolate(R::QQPolyRing, x::Vector{ZZRingElem},
```

Comment:
Wouldn't it make more sense to offers this as a method taking two `Vector{Int}` so it can be used efficiently without conversion? One can then still layer a method taking a `Vector{ZZRingElem}` above it which is implemented in terms of the first method.